src/main/java/com/conveyal/r5/profile/FastRaptorWorker.java

```diff
@@ -628,7 +630,7 @@ public int getWorstCaseFrequencyDepartureTime (TripSchedule schedule, int stopPo
     private void doTransfers (RaptorState state) {
         // avoid integer casts in tight loop below
         int walkSpeedMillimetersPerSecond = (int) (request.walkSpeed * 1000);
-        int maxWalkMillimeters = (int) (request.walkSpeed * request.maxWalkTime * 60 * 1000);
+        int maxWalkMillimeters = walkSpeedMillimetersPerSecond * (request.maxWalkTime * SECONDS_PER_MINUTE);
```

Comment:
So it looks like this one is a pure refactor, we were already applying a distance limit in the transfers.